README.md

```diff
@@ -45,7 +45,9 @@ All country validators are in the "namespace" of the ISO country code.
 | Australia              | AU   | TFN             | Tax/Person/Company | Australian Tax File Number                                                          |
 | Bosnia and Herzegovina | BA   | JMBG            | Person             | Unique Master Citizen Number                                                        |
 | Belize                 | BZ   | TIN             | Person/Company     | Brazilian Tax ID ()                                                                 |
-| Belgium                | BE   | NN              | Person             | Belgian National Number (Numéro National)
+| Belgium                | BE   | BIS             | Person             | Belgian Number for Foreigners                                                       |
+| Belgium                | BE   | INSZ, NISS      | Person             | Belgian Social Security Identification Number (Identificatienummer van de Sociale Zekerheid, Numéro d'Identification Sécurité Sociale) |
```

Comment:
It looks like a Belgian INSZ / NISS can be either a NN (also referred to as NR) or a BIS. See [here](https://archive.wikiwix.com/cache/index2.php?url=https%3A%2F%2Fwww.ksz-bcss.fgov.be%2Ffr%2Fservices-et-support%2Fservices%2Fregistre-national-registres-bcss#federation=archive.wikiwix.com), translated to English: "Within Social Security, the primary key for data exchange is the Social Security Identification Number (NISS) . This NISS is either a National Register (RN) number or a BIS number."

I also updated the `nn` `abbreviations` and removed NISS.